Marlin/ultralcd.cpp

```diff
@@ -3489,6 +3487,7 @@ void kill_screen(const char* lcd_msg) {
       ENCODER_DIRECTION_NORMAL(); \
       if ((int32_t)encoderPosition < 0) encoderPosition = 0; \
       if ((int32_t)encoderPosition > maxEditValue) encoderPosition = maxEditValue; \
+      bool do_it_live = liveEdit;\
```

Comment:
Please remove this line. I thought I got it in the conflict resolution I just did, but apparently not.